Description: Install the dma binary with the correct permissions.
 As per the Debian Policy, change the /usr/sbin/dma permissions from 2555
 to 2755, since removing root write access gains us nothing.
Origin: other: http://svn.ringlet.net/svn/ringlet/mail/dma/
Forwarded: not-needed
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2009-09-01

--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@
 
 BINOWN= root
 BINGRP= mail
-BINMODE=2555
+BINMODE=2755
 WARNS?=	6
 
 # The pmake Debian package does not seem to deal properly with y.tab.h
